Utils/azuretools-core/src/com/microsoft/azuretools/authmanage/AuthMethodManager.java

```diff
@@ -148,6 +149,7 @@ public AzureManager getAzureManager() {
     }
 
     @AzureOperation(name = "account.sign_out", type = AzureOperation.Type.TASK)
+    @CacheEvict() // evict all caches on signing out
```

Comment:
We may keep this as we do need to clean all azure related caches. but some cache is still valid after sign out, for instance, azure reference book meta data.